tests/conftest.py

```diff
@@ -41,7 +41,7 @@
 
 
 if Version(qiskit.__version__) < Version("1.0.0"):
-    test_devices = [AerDevice, BasicAerDevice]
+    test_devices = [AerDevice]
     hw_backends = ["qasm_simulator", "aer_simulator"]
     state_backends = [
```

Comment:
I think all of the `if Version` checks for qiskit in the `conftest.py` file can just be removed now, right? Since the tests shouldn't be running with `Version(qiskit.__version__) < Version("1.0.0")` anymore.

Reply:
Maybe more relevant [in 536](https://github.com/PennyLaneAI/pennylane-qiskit/pull/536) 🤷‍♀️ But just tidying up either way, since it should always use the `else` block once 536 is merged

Reply:
Yeah all the` if Version` checks should be removed in general; there are quite a few littered throughout the tests file as well that I didn't remove to keep this PR lean. 

I can tidy this up in this PR though since it's within the scope of removing the basicaerdevice though.